docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -14,289 +14,286 @@ need to implement it on your own. Don't worry, it's easy!
+    @linter(executable='pylint')
+    class PylintTutorialBear:
+        @staticmethod
+        def create_arguments(filename, file, config_file):
+            pass
 
-The ``output_regex`` is used to group parts of the output (such as ``lines``,
-``columns``, ``severity`` and ``message``) so it can be used by the Lint
-class to yield Results (more on communicating with the user
-:doc:`Writing Bears <Writing_Bears>`).
+``create_arguments`` accepts three parameters:
 
-In order to figure out the ``output_regex`` we have to first see how the
-linter output looks. I will use this file as ``sample.html``
+- ``filename``: The absolute path to the file that gets processed.
+- ``file``: The contents of the file to process, given as a list of lines
+  (including the return character).
+- ``config_file``: The absolute path to a config file to use. If no config file
+  is used, this parameter is ``None``. More on that later.
 
-::
+You can use these parameters to construct the command line arguments. The
+linter expects from you to return an argument sequence here. A tuple is
+preferred. We will do this soon for ``PylintTutorialBear``.
 
-    <html>
-      <body>
-        <h1>Hello, world!</h1>
-      </body>
-    </html>
+So which are the exact command line arguments we need to provide? It depends on
+the output format of the linter. The linter decorator is capable of handling
+different output formats:
 
-Now we can run
+- ``regex``: This parses issue messages yielded by the underlying executable.
+- ``corrected``: Auto-generates results from a fixed/corrected file provided by
+  the tool.
 
-.. Start ignoring LineLengthBear
+In this tutorial we are going to use the ``regex`` output format. But before we
+continue with modifying our bear, we need to figure out how exactly output from
+Pylint looks like so we can parse it accordingly.
 
-::
+We get some promising output when invoking Pylint with
 
-    $ html_lint.py sample.html
-    1:1: Info: Optional Tags: Omit optional tags (optional): You may remove the opening "html" tag.
-    2:3: Info: Optional Tags: Omit optional tags (optional): You may remove the opening "body" tag.
-    4:3: Info: Optional Tags: Omit optional tags (optional): You may remove the closing "body" tag.
-    5:1: Info: Optional Tags: Omit optional tags (optional): You may remove the closing "html" tag.
+::
 
-.. Stop ignoring LineLengthBear
+    $ pylint --msg-template="L{line}C{column}: {msg_id} - {msg}" --reports=n
 
-Our ``output_regex`` should look like this
+Sample output looks like this:
 
 ::
 
-    (line):(column): (severity): (message)
+    No config file found, using default configuration
+    ************* Module coalib.bearlib.abstractions.Linter
+    L1C0: C0111 - Missing module docstring
+    L42C48: E1101 - Class 'Enum' has no 'reverse' member
+    L77C32: E1101 - Class 'Enum' has no 'reverse' member
+    L21C0: R0912 - Too many branches (16/12)
+    L121C28: W0613 - Unused argument 'filename'
```

Comment:
yes! A tutorial shows people how things are done well. If we give them a bad example we get lots of bad bears.